Vegeting/Screens/PostDetail/PostDetailViewController.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  PostDetailViewController.swift
+//  Vegeting
+//
+//  Created by 김원희 on 2022/10/23.
+//
+
+import UIKit
+
+class PostDetailViewController: UIViewController {
+    
+    private lazy var coverImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFill
+        imageView.clipsToBounds = true
+        imageView.image = UIImage(named: "coverImage")
+        return imageView
+    }()
+    
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 18, weight: .bold)
+        label.text = "남미플랜트랩하고 거북이 먹으러 가요"
+        
+        return label
+    }()
+    
+    private lazy var categoryLabel: UILabel = {
+        let label = UILabel()
+        label.text = "맛집"
+        return label
+    }()
+    
+    private lazy var locationLabel: UILabel = {
+        let label = UILabel()
+        label.text = "서울시 동대문구"
+        return label
+    }()
+    
+    private lazy var dateLabel: UILabel = {
+        let label = UILabel()
+        label.text = "12월 19일"
+        return label
+    }()
+    
+    private lazy var capacityLabel: UILabel = {
+        let label = UILabel()
+        label.text = "2/4"
+        return label
+    }()
+    
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.alignment = .center
+        stackView.distribution = .fillProportionally
+        stackView.spacing = 2
+//        stackView.backgroundColor = .blue
+        return stackView
+    }()
+    
+    private lazy var contentTextLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 0
+        label.text = "저번에 너무 가고싶엇는데~~ 긴급휴무로 못 가서 너무 아쉬웠어여. 유명 맛집이라던데 갔다가 옆에 있는 거북이까지 갔다 오면 좋을 것 같아요!! 날짜는 10월 내에 토요일 중에 가능한 날짜 맞춰서 다녀오려고 해요~ㅎㅎ 같이 맛난 점심먹어용!"
+//        label.backgroundColor = .purple
+        return label
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .white
+        
+        setupLayout()
+        setupNavBar()
+    }
+    
+    private func setupNavBar() {
+        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "ellipsis"), style: .done, target: self, action: #selector(showActionSheet))
```

Comment:
p2;
길어지는 함수의 경우 쉼표 다음에 enter를 넣어 가독성을 높이는 게 좋아 보입니다!
```suggestion
navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "ellipsis"),
                                                    style: .done,
                                                    target: self,
                                                    action: #selector(showActionSheet))
```

Reply:
p2;
저는 파라미터 하나씩 끊는게 오히려 보기가 불편한데, 얘기 나눠볼까요?